pom.xml

```diff
@@ -153,12 +153,6 @@
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>pipeline-stage-step</artifactId>
```

Comment:
Anyway this was a `test`-scoped dependency to begin with, which was only needed by the previous version of `SnippetizerTest`.